etcdprobe/etcdprobe.go

```diff
@@ -3,8 +3,8 @@ package etcdprobe
 import (
 	"context"
 
-	etcd "go.etcd.io/etcd/v3/client"
-	errgo "gopkg.in/errgo.v1"
+	etcd "go.etcd.io/etcd/client/v2"
```

Comment:
We probably should use etcd v3:
https://pkg.go.dev/go.etcd.io/etcd/client/v3

Reply:
It was originally designed to target etcd API v2. That's why with this update I'm using the v2 API. But the dependency has been updated to the version which supports Go modules!